### DISCUSSION TITLE
How to get bounding box of image and ops combined

### DISCUSSION THREAD

**Discussion by ShadyHippo** [hajimehoshi/ebiten#2777] [Q&A] [answered]
This might be a fundamentally stupid question, but if I have points on a rectangle before it's rotated, how do I know where those points will be after it's rotated? 

I see that DrawImage is able to get those points out, and I've briefly looked at the code, but let's say I wanted to get the location of two images after they've been drawn to check if they overlap (check collision) How would I do that? Is there something included in engine to do so? I have looked for about an hour and don't see any examples covering this specific request. I can implement myself eventually, but if this is solved already that'd be great. 

Thanks in advance!

**Comment by hajimehoshi** [comment, marked as answer]
I think what you need is [`GeoM.Apply`](https://pkg.go.dev/github.com/hajimehoshi/ebiten/v2#GeoM.Apply), which converts a point to another point by the GeoM.

**Reply by ShadyHippo** [reply]
Thank you for the reply! I think this is exactly what I needed
